resources/packs/gcp/gcp.lr

```diff
@@ -556,43 +559,343 @@ private gcp.storage.bucket @defaults("id") {
   iamConfiguration dict
 }
 
-// GCP Cloud SQL
-gcp.sql {
-  //
-  instances() []gcp.sql.instance
+// GCP Cloud SQL Resources
+private gcp.project.sqlService {
+  // Project ID
+  projectId string
+  // The list of Cloud SQL instances in the current project
+  instances() []gcp.project.sqlService.instance
 }
 
 // GCP Cloud SQL Instance
-private gcp.sql.instance @defaults("name") {
-  // Instance name
-  name string
+private gcp.project.sqlService.instance @defaults("name") {
+  // Project ID
+  projectId string
+  // All maintenance versions applicable on the instance
+  availableMaintenanceVersions []string
   // Backend type
   backendType string
-  // Connection name
+  // Connection name of the instance used in connection strings
   connectionName string
-  // Database version
+  // Creation timestamp
+  created time
+  // The current disk usage of the instance in bytes. This is deprecated, monitoring should be used instead
+  currentDiskSize int
+  // The current database version running on the instance
+  databaseInstalledVersion string
+  // The database engine type and version
   databaseVersion string
-  // GCE zone
+  // Disk encryption configuration
+  diskEncryptionConfiguration gcp.project.sqlService.instance.diskEncryptionConfiguration
+  // Disk encryption status
+  diskEncryptionStatus gcp.project.sqlService.instance.diskEncryptionStatus
+  // The name and status of the failover replica
+  failoverReplica gcp.project.sqlService.instance.failoverReplica
+  // The Compute Engine zone that the instance is currently service from
   gceZone string
   // Instance type
   instanceType string
-  // Kind
-  kind string
-  // Current disk size
-  currentDiskSize int
-  // Maximum disk size
+  // The assigned IP addresses
+  ipAddresses []gcp.project.sqlService.instance.ipMapping
+  // The current software version on the instance
+  maintenanceVersion string
+  // The name of the instance which will act as primary in the replica
+  masterInstanceName string
+  // The maximum disk size in bytes
   maxDiskSize int
-  // Instance state
-  state string
+  // Instance name
+  name string
   // Project
   // TODO: should this be id or name
   project string 
   // Region
   region string
+  // Replicas
+  replicaNames []string
+  // Settings
+  settings gcp.project.sqlService.instance.settings
   // Service account email address
   serviceAccountEmailAddress string
-  // Settings
-  settings dict
+  // Instance state
+  state string
+  // List of the databases in the current SQL instance
+  databases() []gcp.project.sqlService.instance.database
+}
+
+// GCP Cloud SQL Instance database
+private gcp.project.sqlService.instance.database @defaults("name") {
+  // Project ID
+  projectId string
+  // Charset value
+  charset string
+  // Collation
+  collation string
+  // Name of the Cloud SQL instance
+  instance string
+  // Name of the database
+  name string
+  // Project
+  project string
```

Comment:
I think this should be the `projectId`. Going forward we may want to use `.project()` to get the proper resource.

Reply:
Now `projectId` and `project` are identical. I left the `project` because it was already there from previous changes. I can remove it if that's fine

Reply:
oh wait that's the other resource. I will remove it from this one